src/meta/meta_http_service.cpp

```diff
@@ -611,6 +610,61 @@ void meta_http_service::query_duplication_handler(const http_request &req, http_
     resp.body = duplication_query_response_to_string(rpc_resp);
 }
 
+void meta_http_service::start_bulk_load_handler(const http_request &req, http_response &resp)
+{
+    if (!redirect_if_not_primary(req, resp)) {
+        return;
+    }
+
+    if (_service->_bulk_load_svc == nullptr) {
+        resp.body = "bulk load is not enabled";
+        resp.status_code = http_status_code::not_found;
+        return;
+    }
+
+    start_bulk_load_request request;
+    bool ret = json::json_forwarder<start_bulk_load_request>::decode(req.body, request);
+    if (!ret) {
+        resp.body = "invalid request structure";
+        resp.status_code = http_status_code::bad_request;
+        return;
+    }
+    if (request.app_name.empty()) {
+        resp.body = "app_name should not be empty";
+        resp.status_code = http_status_code::bad_request;
+        return;
+    }
+    if (request.cluster_name.empty()) {
+        resp.body = "cluster_name should not be empty";
+        resp.status_code = http_status_code::bad_request;
+        return;
+    }
+    if (request.file_provider_type.empty()) {
+        resp.body = "file_provider_type should not be empty";
+        resp.status_code = http_status_code::bad_request;
+        return;
+    }
+    if (request.remote_root_path.empty()) {
+        resp.body = "remote_root_path should not be empty";
+        resp.status_code = http_status_code::bad_request;
+        return;
+    }
```

Comment:
Can this simple args validate put client side?

Reply:
In current implmetation, client and server both will validate arguments.

Reply:
I will implement HTTP argument validation in later PR. I actually did it in my staging branch. But haven't ready for review yet. Got no time to make it steady.